### ISSUE TITLE
Read before you open an issue with broken skins

### ISSUE THREAD

**Issue by 6oody** [R3nzTheCodeGOD/R3nzSkin#1]
this shows in the real skin
![image](https://user-images.githubusercontent.com/42409047/127751380-e6ab655e-d131-4f48-b1e2-bd8ec19e99a2.png)
and in the R3nskin
![image](https://user-images.githubusercontent.com/42409047/127751403-f1bc0926-e62b-4d7f-8e4e-2fb75057495f.png)
i really hope u add this in kayn's skins

**Comment by R3nzTheCodeGOD** [comment]
@AveAcVale Nidalee, Elise, etc... It is possible to fix the second forms of characters, but fixing this issue will cause crashes in the middle of the game.

**Reason:** Constantly editing the `CharacterDataStack` structure in the game causes the structure to become unstable and the game crashes.

**Comment by RushWTF** [comment]
Project Renekton, the head are hide, you can only see the head for a seconds in recall animation, i dont know if this problem happens because the head need a serverside interaction.

**Comment by 04255** [comment]
the Butcher of the Sands  Renekton
PROJECT T2 skin Lost head.
I know. It's funny

**Comment by 04255** [comment]
> Renekton项目，头部是隐藏的，你只能在召回动画中看到头部几秒钟，我不知道这个问题是否因为头部需要服务器端交互而发生。

ctrl+1234 Can make your head return briefly

**Comment by alphaaced** [comment]
chosen master yi sword does not appear.

**Comment by kudanovaa** [comment]
why is the voice acting in the new legendary skin on kaisu old and not changed?

**Comment by N3TR1X** [comment]
odyssey kayn e and recall, on original skins his E and Recall cast animations different

**Comment by omaruo** [comment]
Odyssey Kayn (R)

**Comment by omaruo** [comment]
#518

**Comment by meeezaezaa** [comment]
galaxy slayer zed Q animation not right and the shadow emotion isnt right if you W Q

**Comment by Reraz** [comment]
Neeko has problem changing character, she changes to a random skin character.

**Comment by Reraz** [comment]
Debonair Zed shadow doesn't move using gestures and the effects are not right.

**Comment by Reraz** [comment]
Battle academia Caitlyn, the passive effect is wrong.

**Comment by Furtass** [comment]
Dawn Bringer Riven R Bugged

**Comment by Momen-17** [comment]
Empyrean Pyke ult bugged

**Comment by Reraz** [comment]
Kayn W dont move on blue form.

**Comment by Reraz** [comment]
Kayn W blue form dont move.

**Comment by Reraz** [comment]
Kayn W blue form dont move.

**Comment by eternal-art** [comment]
> @AveAcVale Nidalee, Elise, etc... It is possible to fix the second forms of characters, but fixing this issue will cause crashes in the middle of the game.
> 
> **Reason:** Constantly editing the `CharacterDataStack` structure in the game causes the structure to become unstable and the game crashes.

this is true. The best example of an unstable game is using any skin for illaoi. When you hit someone with E and their spirit is drawn to you then a Lag/Freeze of the game occurs. The same happens when you use R. Let's hope this code on github doesn't spread to a larger audience and get it banned.

**Comment by Reraz** [comment]
Weird i used illaoi and i had no problem.

**Comment by Reraz** [comment]
Blue kayn shadow doesnt move using W.

**Comment by Reraz** [comment]
Senna winterblessed the final effect when she disappears. The effect doesnt work.

**Comment by Reraz** [comment]
Viego's abilities dont match the skin.

**Comment by Reraz** [comment]
Viego's R and passive dont match the skin.

**Comment by Reraz** [comment]
Syndra's orbs dont change with skin.

**Comment by Reraz** [comment]
there is a problem with summons, for example w zyra , maoikay E, teemo R. Annie R. These used to work before the update.

**Comment by eleeyn** [comment]
there is currently a problem with the particles (for example, nunu w, orianna ball, annie r.) it used to work on the previous release.

**Comment by R3nzTheCodeGOD** [comment]
> there is currently a problem with the particles (for example, nunu w, orianna ball, annie r.) it used to work on the previous release.

1de72084f64e2c39a63a71e5abd7f3103f9421e3 should work now

**Comment by slayer99234** [comment]
Galaxy slayer zed q flick not correct ( the shadows q flick are correct)

**Comment by CountryballsQer** [comment]
Mythmaker irelia doesnt have specail voicelines in game
